services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,28 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to sync meters for unmetered flow"() {
+        given: "A flow with custom meter rate and burst, that differ from defaults"
+        def randomSwitches = topology.activeSwitches
```

Comment:
This is fine, but I don't quite understand the choice of variable name, there's nothing random here.